deploy/v2.0.0-alpha.1/rbac-csi-azuredisk-scheduler-extender.yaml

```diff
@@ -51,7 +51,7 @@ rules:
     verbs: ["get", "list", "watch"]
   - apiGroups: [""]
     resources: ["persistentvolumeclaims"]
-    verbs: ["get", "list", "watch"] # Add ["patch", "update"] for CSIMigration support
+    verbs: ["get", "list", "watch", "update"] # Add ["patch", "update"] for CSIMigration support
```

Comment:
You should update the comment to reflect that "update" permission is required for regardless of whether CSI migration is enabled.